classes/class-settings.php

```diff
@@ -270,13 +270,7 @@ public function add_display_name_search_columns( $search_columns, $search, $quer
 	public function get_option_key() {
 		$option_key = $this->option_key;
 
-		$current_page = wp_stream_filter_input( INPUT_GET, 'page' );
-
-		if ( ! $current_page ) {
-			$current_page = wp_stream_filter_input( INPUT_GET, 'action' );
-		}
-
-		if ( 'wp_stream_network_settings' === $current_page ) {
```

Comment:
Use the network settings if the plugin is network-activated. Is there a situation we would like to use network settings when the plugin is _not_ network activated?